tfx/components/example_gen/base_example_gen_executor.py

```diff
@@ -40,18 +40,33 @@
 DEFAULT_FILE_NAME = 'data_tfrecord'
 
 
-def _ExamplePartitionKey(record: tf.train.Example,
-                         split_config: example_gen_pb2.SplitConfig) -> bytes:
-  """Generates key for partition for tf.train.Example."""
+def _GeneratePartitionKey(
+    record: Union[tf.train.Example, tf.train.SequenceExample, bytes],
+    split_config: example_gen_pb2.SplitConfig
+) -> bytes:
+  """Generates key for partition."""
 
   if not split_config.HasField('partition_feature_name'):
+    if isinstance(record, bytes):
+      return record
     return record.SerializeToString(deterministic=True)
 
+  features = ""
```

Comment:
you don't need this (and feature is not str, I guess you mean feature=None), python's scope is different, you can access the var defined in if block

Reply:
got it, thanks for the catch!